src/AppBundle/PullRequests/Repository.php

```diff
@@ -127,14 +111,33 @@ public function getCommentsByExpressionFrom(
         $userComments = $this->getCommentsFrom($pullRequest, $userLogin);
 
         foreach ($userComments as $userComment) {
-            if (strpos($userComment->getBody(), $expression)) {
+            if (strpos($userComment->getBody(), $expression) !== false) {
```

Comment:
and this is the bug: if the expression is in position 0, the comment wasn't found.